.github/workflows/ci.yml

```diff
@@ -55,6 +55,8 @@ jobs:
       - uses: codecov/codecov-action@v4
         if: matrix.version == '1' && matrix.os == 'ubuntu-latest' && matrix.arch == 'x64'
         with:
+          fail_ci_if_error: true
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
The token has to be retrieved from the codecov website and added as a Github secret.